modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -1275,10 +1275,12 @@ CV__DNN_INLINE_NS_BEGIN
           *  @param[out] boxes A set of bounding boxes.
           *  @param[in] confThreshold A threshold used to filter boxes by confidences.
           *  @param[in] nmsThreshold A threshold used in non maximum suppression.
+          *  @param[in] nmsAcrossClasses Determines if non maximum suppression should be applied when labels are different.
           */
          CV_WRAP void detect(InputArray frame, CV_OUT std::vector<int>& classIds,
                              CV_OUT std::vector<float>& confidences, CV_OUT std::vector<Rect>& boxes,
-                             float confThreshold = 0.5f, float nmsThreshold = 0.0f);
+                             float confThreshold = 0.5f, float nmsThreshold = 0.0f,
+                             bool nmsAcrossClasses = false);
```

Comment:
Please make as a method:
```cpp
CV_WRAP DetectionModel& setNMSAcrossClasses (bool value);
```
Otherwise this method is too complicated.

Reply:
Hi, I'm doing this, however for when the return type is DetectionModel&, I get an error in the python binding that no default contructor for DetectionModel exists. It works fine when the type is void for the setter though.

Reply:
Yes, please take a look at Model class definition: https://github.com/opencv/opencv/blob/master/modules/dnn/include/opencv2/dnn/dnn.hpp

Reply:
Right, just saw the 'need to fix bindings first' comment. Does that mean I should leave it as void for now?